localdev/docker-compose.yml

```diff
@@ -22,18 +22,19 @@ services:
     user: "0"
     ports:
       - "9092:9092"
+      - "9093:9093"
     environment:
       # for configuration options see https://github.com/bitnami/containers/blob/main/bitnami/kafka/README.md#configuration
       # and prefix all configs from https://kafka.apache.org/documentation/#adminclientconfigs with KAFKA_CFG_
       KAFKA_CFG_BROKER_ID: "1"
       KAFKA_ENABLE_KRAFT: "yes"
       ALLOW_PLAINTEXT_LISTENER: "yes"
-      KAFKA_CFG_LISTENER_SECURITY_PROTOCOL_MAP: 'CONTROLLER:PLAINTEXT,DOCKER:PLAINTEXT,HOST:PLAINTEXT'
-      KAFKA_CFG_ADVERTISED_LISTENERS: 'DOCKER://kafka:29092,HOST://localhost:9092'
+      KAFKA_CFG_LISTENER_SECURITY_PROTOCOL_MAP: 'CONTROLLER:PLAINTEXT,DOCKER:PLAINTEXT,HOST:PLAINTEXT,CLI:PLAINTEXT'
+      KAFKA_CFG_ADVERTISED_LISTENERS: 'DOCKER://kafka:29092,HOST://localhost:9092,DOCKERCLI://host.docker.internal:9093'
```

Comment:
:thinking: I think if you attach the backup cli container to the host network this configuration is obsolete.

e.g. `docker run --network host lovelysystems/lovely-kafka-backup:dev restore ...`

Reply:
Use this as an example